spec/acceptance/lastEvaluatedKeyWithFilterAndPagingTest.spec.ts

```diff
@@ -229,13 +246,31 @@ describe('lastEvaluatedKeyWithFilterAndPagingTest', function ()
        assert.deepEqual(page1.items[0], user2);
        assert.deepEqual(page1.items[1], user6);
        assert.deepEqual(page1.items[2], user7);
-       assert.isDefined(page1.lastEvaluatedKey)
+       assert.isDefined(page1.firstEvaluatedKey);
+       assert.isDefined(page1.lastEvaluatedKey);
 
        let page2 = await nodenamo.list().from(User).filter(filter).limit(3,3).resume(page1.lastEvaluatedKey).execute<User>();
 
        assert.equal(page2.items.length, 1);
        assert.deepEqual(page2.items[0], user8);
+       assert.isDefined(page2.firstEvaluatedKey);
        assert.isUndefined(page2.lastEvaluatedKey);
+
+       let page1Again = await nodenamo.list().from(User).filter(filter).limit(3,3).order(false).resume(page2.firstEvaluatedKey).execute<User>();
+       assert.deepEqual(page1Again.items, [
+           user7,
+           user6,
+           user2
+       ]);
+       assert.isDefined(page1Again.firstEvaluatedKey);
+       assert.isDefined(page1Again.lastEvaluatedKey);
+
+       let page2Again = await nodenamo.list().from(User).filter(filter).limit(3,3).resume(page1Again.firstEvaluatedKey).execute<User>();
+       assert.deepEqual(page2Again.items, [
+           user8
+       ]);
+       assert.isDefined(page2Again.firstEvaluatedKey);
+       assert.isUndefined(page2Again.lastEvaluatedKey);
```

Comment:
Can we assert that page2Again.firstEvaluatedKey and page2Again.lastEvaluatedKey are equal to page2.firstEvaluatedKey and page2.lastEvaluatedKey respectively?